sentinel-transport/sentinel-transport-simple-http/src/main/java/com/alibaba/csp/sentinel/transport/command/SimpleHttpCommandCenter.java

```diff
@@ -182,13 +175,23 @@ public void stop() throws Exception {
 
         @Override
         public void run() {
+            //loop accept ,receive request and
             while (true) {
                 Socket socket = null;
                 try {
                     socket = this.serverSocket.accept();
                     setSocketSoTimeout(socket);
+                    //Give it to the biz threadPool for processing
                     HttpEventTask eventTask = new HttpEventTask(socket);
-                    bizExecutor.submit(eventTask);
+                    //add server initiative timeout，prevent bizThreadPool full normal requests may be affected
+                    Future<String> future = bizExecutor.submit(eventTask,"ok");
+                    try {
+                        future.get(DEFAULT_SERVER_SO_TIMEOUT, TimeUnit.MILLISECONDS);
+                    }catch (TimeoutException | InterruptedException | ExecutionException ex){
```

Comment:
Could you please reformat your code?

Reply:
I'll try，I always use Eclipse Code Formatter auto format when save the code file。The IDE has just been upgraded, may not work